src/main/resources/assets/controlify/lang/fr_fr.json

```diff
@@ -0,0 +1,393 @@
+  "controlify.binding.controlify.gui_navi_left": "Navigation dans l'interface - Gauche",
+  "controlify.binding.controlify.gui_navi_right": "Navigation dans l'interface - Droite",
+  "controlify.binding.controlify.cycle_opt_forward": "Avancer dans les options",
+  "controlify.binding.controlify.cycle_opt_backward": "Reculer dans les options",
+  "controlify.binding.controlify.clear_binding": "Effacer la liaison"
+
+  "controlify.binding_category.gui": "Interface utilisateur (GUI)",
+  "controlify.binding_category.vmouse": "Souris virtuelle",
+
+  "controlify.custom_binding.vanilla_description": "Liaison de touches générée automatiquement à partir de l'API Fabric."
+
+  "controlify.guide.ingame.inventory": "Ouvrir l'inventaire",
+  "controlify.guide.ingame.swim_up": "Nager vers le haut",
+  "controlify.guide.ingame.start_elytra": "Ouvrir les ailes de l'Élytre",
+  "controlify.guide.ingame.fly_up": "Monter en volant",
+  "controlify.guide.ingame.fly_down": "Descendre en volant",
+  "controlify.guide.ingame.start_sneaking": "Commencer à s'accroupir",
+  "controlify.guide.ingame.stop_sneaking": "Arrêter de s'accroupir",
+  "controlify.guide.ingame.start_swimming": "Commencer à nager",
+  "controlify.guide.ingame.start_sprinting": "Commencer à sprinter",
+  "controlify.guide.ingame.stop_swimming": "Arrêter de nager",
+  "controlify.guide.ingame.stop_sprinting": "Arrêter de sprinter",
+  "controlify.guide.ingame.sneak": "S'accroupir",
+  "controlify.guide.ingame.dismount": "Démonter",
+  "controlify.guide.ingame.swim_down": "Nager vers le bas",
+  "controlify.guide.ingame.drop": "Lâcher l'objet",
+  "controlify.guide.ingame.swap_hands": "Changer de main",
+  "controlify.guide.ingame.attack": "Attaquer",
+  "controlify.guide.ingame.break": "Casser",
+  "controlify.guide.ingame.use": "Utiliser",
+  "controlify.guide.ingame.spectate": "Spectateur",
+  "controlify.guide.ingame.interact": "Interagir",
+  "controlify.guide.ingame.pick_block": "Sélectionner un bloc"
+
+  "controlify.guide.container.place_all": "Placer tout",
+  "controlify.guide.container.place_one": "Placer",
+  "controlify.guide.container.swap": "Échanger",
+  "controlify.guide.container.drop": "Lâcher",
+  "controlify.guide.container.take": "Prendre",
+  "controlify.guide.container.exit": "Sortir",
+  "controlify.guide.container.take_half": "Prendre la moitié",
+  "controlify.guide.container.take_one": "Placer un",
+  "controlify.guide.container.quick_move": "Déplacement rapide"
+
+  "controlify.calibration.title": "Calibration du contrôleur pour '%s'",
+  "controlify.calibration.info": "Cette calibration optimise les paramètres de votre contrôleur pour éliminer le drift des sticks et créer une lecture précise du gyroscope (si votre contrôleur le supporte). Avant de commencer, assurez-vous de poser votre contrôleur sur une surface plane et de ne pas toucher les sticks.\n\nCela ne prendra que quelques secondes.",
```

Comment:
idem : manette est le terme utilisé jusqu’ici.

Reply:
I'd like to point out that there was a pause in the middle of this translation, perhaps that's why I only used the word "Contrôleur" from this point onwards, please excuse me.

Modified.